src/backend/executor/process.ts

```diff
@@ -206,8 +207,10 @@ export class ExecutorManager implements Disposable {
 
     private executionPriority = [
         ProcessType.version,
+        ProcessType.checkers,
         ProcessType.parse,
-        ProcessType.analyze
+        ProcessType.log,
```

Comment:
Is this ProcessType.log new? If i am correct this is related to the Codechecker log command.
Can you please mention in the commit message, that you added this feature?

Reply:
It's been there previously, the only thing I did here is put it into the execution priority list. The only new one is ProcessType.checkers (which is now noted).